src/jit/flowgraph.cpp

```diff
@@ -21721,6 +21742,58 @@ void Compiler::fgDebugCheckBlockLinks()
     }
 }
 
+//------------------------------------------------------------------------------
+// fgDebugCheckNodesUniqueness: Check that each tree in the method has its own unique gtTreeId.
+//
+void Compiler::fgDebugCheckNodesUniqueness()
+{
+    unsigned     numberOfTrees = JitTls::GetCompiler()->compGenTreeID;
```

Comment:
`JitTls::GetCompiler()` seems odd, this function is a member of `Compiler`.

Reply:
Fixed